packages/codemods/src/modRunner/tests/upgrade.test.ts

```diff
@@ -0,0 +1,49 @@
+import { _upgradeTest } from '../../upgrade';
+import * as path from 'path';
+import * as fs from 'fs-extra';
+import * as os from 'os';
+import { getTsConfigs } from '../runnerUtilities';
+
+const tempDir = os.tmpdir();
+const tempPath = path.join(tempDir, 'codemods');
+const startingPath = process.cwd();
+
+describe('modRunner tests', () => {
+  it('saves syncronously correctly', () => {
+    const saveAsync = jest.fn();
+    const saveSync = jest.fn();
+    try {
+      const temp = fs.mkdtempSync(tempPath);
```

Comment:
It's probably a good idea to clean this up in `finally` (or move the temp dir creation/cleanup to a beforeEach/afterEach). Temp directories do get automatically cleaned up eventually but not always very quickly.

Reply:
I added a `beforeEach` and `afterEach` to handle the folder cleanup and creation. Good idea!